_sass/fonts.scss

```diff
@@ -8,7 +32,6 @@
          url('../fonts/OpenSans-Light-webfont.svg#OpenSansLight') format('svg');
```

Comment:
Don't these font files also need to be added to the `assets/fonts` directory?

Reply:
They are automatically added during the build. I don't know why, though. I can add them if it doesn't work on your machine.

Reply:
Well, you're adding the Source Code Pro files to that folder in this PR, so it would make sense for these to be there as well, if only for consistency.

Reply:
Done!